### DISCUSSION TITLE
Date picker bugged.

### DISCUSSION THREAD

**Discussion by tokkisva** [Budibase/budibase#8559] [Help] [answered]
Hi All,

No matter what date I set on the date picker within any Budibase screen, it defaults to 'today' (11/7/2022). I can't choose a date before or after. Tried this in Firefox and Edge, same issue.

![image](https://user-images.githubusercontent.com/38924262/200336168-98a94bec-5ca1-4ecd-bbd0-a6bb0bb938a0.png)
Thoughts? This even happens when editing a column's 'Earliest/Latest' date. 

Version 2.1.9

**Comment by melohagan** [comment, marked as answer]
Hey @tokkisva 

Could you please upgrade to the latest version: **v2.1.15**

This bug was fixed: https://github.com/Budibase/budibase/issues/8487
